src/encoded/types/file.py

```diff
@@ -139,7 +139,9 @@ def __name__(self):
     def unique_keys(self, properties):
         keys = super(File, self).unique_keys(properties)
         if properties.get('status') != 'replaced':
-            if 'md5sum' in properties:
+            file_output_category = self.schema['output_type_output_category'].get(properties.get('output_type'))
```

Comment:
Are you sure you can't properties.get('output_category')?

Reply:
replaced by the calculated 'output_category'